conversation.go

```diff
@@ -0,0 +1,131 @@
+package podio
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"path"
+	"strconv"
+)
+
+// Conversation holds meta-data about a group or direct chat session
+type Conversation struct {
+	ConversationId uint      `json:"conversation_id"`
+	Reference      Reference `json:"ref"`
+	CreatedOn      Time      `json:"created_on"`
+	CreatedBy      ByLine    `json:"created_by"`
+
+	Excerpt      string   `json:"excerpt"`
+	Starred      bool     `json:"starred"`
+	Unread       bool     `json:"unread"`
+	UnreadCount  uint     `json:"unread_count"`
+	LastEvent    Time     `json:"last_event_on"`
+	Subject      string   `json:"subject"`
+	Participants []ByLine `json:"participants"`
+	Type         string   `json:"type"` // direct or group
+}
+
+// ConversationEvent is a single message from a sender to a conversation
+type ConversationEvent struct {
+	EventID uint `json:"event_id"`
+
+	Action string `json:"action"`
+	Data   struct {
+		MessageID uint          `json:"message_id"`
+		Files     []interface{} `json:"files"` // TODO: add structure
+		Text      string        `json:"text"`
+		EmbedFile interface{}   `json:"embed_file"` // TODO: add structure
+		Embed     interface{}   `json:"embed"`      // TODO: add structure
+		CreatedOn Time
+	}
+
+	CreatedVia struct {
+		ID         uint   `json:"id"`
+		URL        string `json:"url"`
+		AuthClient uint   `json:"auth_client"`
+		Display    bool   `json:"display"`
+		Name       string `json:"name"`
+	} `json:"created_via"`
```

Comment:
Could [`Via`](https://github.com/andreas/podio-go/blob/a5a3941d114d31a5dc6c5b9e4c1f6aac6f8a1327/byline.go#L18) be reused?